### ISSUE TITLE
Extra Marrital Affair - missing more affairs than 1

### ISSUE THREAD

**Issue by erikk-avans** [hansvana/ML5_NN_Trainer#30]
Classification multi-class of # of affairs
All columns selected except index
target is nbaffairs

Issue : prediction is always nbaffairs-1 or nbaffairs-0 - it does now come up with any nbaffairs-2, nbaffairs-3, nbaffairs--7 and nbaffairs-12

**Comment by hansvana** [comment]
I don't believe this to be a bug. If you look at the data nbaffairs is almost always 0, with no correlation to any outliers. Which is why the NN is giving me a loss of about 0.6, and a 0.5 to 0.8 confidence that the result of any input is always 0.

**Comment by erikk-avans** [comment]
You might use an older version of the dataset. Please fetch it again from the Github. 
Main drivers for marrital affair : sex,  child, age, ym

quick test to reproduce the issue : set ym=18; there is 80% chance that you have an affair according the data. The trainer does not show any.
